### PR TITLE
[release-1.1] libct/cg: IsCgroup2HybridMode: don't panic

### PR DESCRIPTION
**Pull request by mrunalp**
In case statfs("/sys/fs/cgroup/unified") fails with any error other
than ENOENT, current code panics. As IsCgroup2HybridMode is called from
libcontainer/cgroups/fs's init function, this means that any user of
libcontainer may panic during initialization, which is ugly.

Avoid panicking; instead, do not enable hybrid hierarchy support and
report the error (under debug level, not to confuse anyone).

Basically, replace the panic with "turn off hybrid mode support"
(which makes total sense since we were unable to statfs its root).

Signed-off-by: Kir Kolyshkin <kolyshkin@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@AkihiroSuda @thaJeztah @kolyshkin I picked this into 1.1 so we could tag a release there to vendor into k8s.

cc: @liggitt

**Review by AkihiroSuda — APPROVED** [review]

**Review by kolyshkin — APPROVED** [review]
LGTM
